packages/platform/atoms/availability/AvailabilitySettings.tsx

```diff
@@ -507,23 +508,17 @@ export function AvailabilitySettings({
                 )}
               </div>
             </div>
-            {!isPlatform ? (
-              <div className="border-subtle my-6 rounded-md border">
-                {schedule?.workingHours && (
-                  <DateOverride
-                    workingHours={schedule.workingHours}
-                    userTimeFormat={timeFormat}
-                    travelSchedules={travelSchedules}
-                    weekStart={
-                      ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"].indexOf(
-                        weekStart
-                      ) as 0 | 1 | 2 | 3 | 4 | 5 | 6
-                    }
-                  />
-                )}
-              </div>
-            ) : (
-              <></>
+            {enableOverrides && (
```

Comment:
`enableOverrides` prop allows users to enable or disable date overrides inside an atom